app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -140,72 +141,86 @@ class FollowUsers extends Component {
-              <h1 className="title">Suggested people to follow</h1>
-              <h2 className="subtitle">Let&apos;s review a few things first</h2>
-            </header>
-
-            <div
-              data-testid="onboarding-users"
-              className="onboarding-modal-scroll-container"
-            >
-              {users.map((user) => (
-                <button
-                  data-testid="onboarding-user-button"
-                  type="button"
-                  onClick={() => this.handleClick(user)}
-                  onKeyDown={() => this.handleKeyDown(user)}
-                  className={
-                    selectedUsers.includes(user)
-                      ? 'user content-row selected'
-                      : 'user content-row unselected'
-                  }
-                >
-                  <figure className="user-avatar-container">
-                    <img
-                      className="user-avatar"
-                      src={user.profile_image_url}
-                      alt="profile"
-                    />
-                  </figure>
-                  <div className="user-info">
-                    <h4 className="user-name">{user.name}</h4>
-                    <p className="user-summary">
-                      {he.unescape(user.summary || '')}
-                    </p>
-                  </div>
-                  <button
-                    data-testid="onboarding-user-following-status"
-                    type="button"
-                    className="user-following-status"
-                  >
-                    {selectedUsers.includes(user) ? 'Following' : 'Follow'}
-                  </button>
-                </button>
-              ))}
+      <Focustrap>
+        <div
+          data-testid="onboarding-follow-users"
+          className="onboarding-main crayons-modal"
+        >
+          <div
+            className="crayons-modal__box overflow-auto"
+            role="dialog"
+            aria-labelledby="title"
+            aria-describedby="subtitle"
+          >
+            <Navigation
+              prev={prev}
+              next={this.handleComplete}
+              canSkip={canSkip}
+              slidesCount={slidesCount}
+              currentSlideIndex={currentSlideIndex}
+            />
+            <div className="onboarding-content toggle-bottom">
+              <header className="onboarding-content-header">
+                <h1 id="title" className="title">
+                  Suggested people to follow
+                </h1>
+                <h2 id="subtitle" className="subtitle">
+                  Let&apos;s review a few things first
+                </h2>
+              </header>
+
+              <ul
+                data-testid="onboarding-users"
+                className="onboarding-modal-scroll-container"
+              >
+                {users.map((user) => (
+                  <li key={user.name}>
+                    <button
```

Comment:
I think this button should have tabIndex="-1".
I think we only want the follow button to be in the tab order.

Reply:
Here is the flow with the tabindex -1

![forem__outer-button](https://user-images.githubusercontent.com/3534427/95470634-ef460100-0978-11eb-98a0-0d77662bc3c7.gif)

only 1 tab to get to the next follow button

Reply:
Thank you @Link2Twenty. But there is one issue though the screen reader is not reading out items in the list so the user (the name of the user and the description below now) it just says follow. This might cause confusion for users who are using screen readers since they don't know which user did they just followed. I can add **aria-label** to the follow button saying 
**"Follow Jere Thompson"** it will help but it will still miss out the description but I don't know if that is ok.

Reply:
When I remove the handler ```this.handleClick``` it is not happening I did try adding both ```event.preventDefault()``` and ```event.stopPropagation()``` they don't seem to affect this. It is something to do with the state change. Let me keep searching.

Reply:
It is a bug in the ```FocusTrap``` component that is causing this when I remove it is working properly. Let me keep searching further.

Reply:
I was able to fix this by moving FocusTrap component one level up to **app/javascript/onboarding/Onboarding.jsx** there are some tests failing (even though what the test is looking for is working when tested manually) let me fix those too.

Reply:
I have fixed the breaking tests too

Reply:
To answer your comment regarding the button name, @Rafi993, an `aria-label` would help to uniquely identify the individual follow buttons. That would solve an issue that exists throughout the site where controls are repeated (i.e. "Links with the same name have a similar purpose" in the axe extension).

Reply:
Thank you @marcysutton I have made the change. I did not know about this before.